src/components/Epr.js

```diff
@@ -15,16 +16,18 @@ export default class Epr extends React.Component {
         this.setState({ locations });
     }
 
-    addSupplement(supplement) {
-        const { supplements } = this.state;
-        supplements.push(supplement);
-        this.setState({ supplements });
+    updateSupplements(supplements) {
+        // I figured it would be cleaner, since supplements can theoretically contain quite a bit of information, to have a dictionary rather than an array
+        // If we want to access the values as an array, of course we can just do Object.values or similar
+        // I'm also doing most of the management (adding and deleting individual supplements) in the form itself. 
+        // So the update only gets pushed up to the global state when they move on to the next section
```

Comment:
I'm doing this too (only pushing to the global state when they move onto the next section) in the basic info page so that should be great!